Taylor and Francis+NEJM.js

```diff
@@ -161,7 +161,7 @@ function finalizeItem(item, doc, doi, baseUrl) {
 	let sectionheading = ZU.xpathText(doc, '//div[@class="toc-heading"]');
 	if (sectionheading) {
 		sectionheading = sectionheading.trim();
-		if (sectionheading.match(/^(Book )?(Reviews|Symposium)?/i))
+		if (sectionheading.match(/^(Book )?(Reviews|Symposium)?$/i))
```

Comment:
This will not solve the problem => the regex will match any string, just like before